rules/naming/src/Guard/BreakingVariableRenameGuard.php

```diff
@@ -118,10 +120,13 @@ public function shouldSkipVariable(
         return $this->isUsedInIfAndOtherBranches($variable, $currentName);
     }
 
-    public function shouldSkipProperty(Property $property, string $currentName): bool
+    public function shouldSkipProperty(PropertyRename $propertyRename): bool
     {
-        $propertyType = $this->nodeTypeResolver->resolve($property);
-        return $this->isDateTimeAtNamingConvention($propertyType, $currentName);
+        if ($this->isRamseyUuidInterface($propertyRename->getProperty())) {
+            return true;
+        }
+
+        return $this->isDateTimeAtNamingConvention($propertyRename->getProperty());
```

Comment:
I wanted to pass ValueObject into these methods as well, it makes, perfectly, sense for `isDateTimeAtNamingConvention` but to make this I need to make some additional changes in RenameParamToMatchTypeRector.php.

If it makes to you (to pass ValueObect to these methods) I'd prefer to do this in separate PR.

Reply:
> If it makes to you (to pass ValueObect to these methods) I'd prefer to do this in separate PR.

Sure :+1: 

Yet it should be somehow standardize in naming-related method. I was actually surprised, that property name resolved once into object and once into string.